### ISSUE TITLE
Possible problem with the onnx representation

### ISSUE THREAD

**Issue by silent-code** [dusty-nv/jetson-inference#736]
To test the onnx intermediate representation, I converted our original base model, **mobilenet-v1-ssd-mp-0_675.pth** to onnx with the appropriate command:

python3 onnx_export.py --input="./models/model_orig/mobilenet-v1-ssd-mp-0_675.pth" --model-dir=models/model_orig

and included the original PASCAL VOC labels in label.txt, including the BACKGROUND class, since these are the classes it was trained on [here](https://github.com/qfgaohao/pytorch-ssd).

The problem is that this network will then classify everything as class 1 on a video. This issue also exists when re-training the base net **mobilenet-v1-ssd-mp-0_675.pth** on a custom dataset and converting to onnx. I believe @JaredP94 is experiencing a similar [issue](https://github.com/dusty-nv/jetson-inference/issues/715) as well.

Note that running ssd-inception-v2 on the same video will produce the correct, multiple classes.

Any advice? 

Thanks again, in advance.
Regards, Tim

**Comment by JaredP94** [comment]
Hi @silent-code, curious to know what your issue was? :)

**Comment by silent-code** [comment]
@JaredP94 

I seem to be having a similar issue as you where everything with my re-trained network classifies with the class 1 label from the list contained in label.txt:

BACKGROUND
class 1 

I thought maybe something was getting messed up in the onnx conversion step so I tested simply converting the base network, **mobilenet-v1-ssd-mp-0_675.pth** to onnx without doing any re-trainning. It appears to work correctly, classifying multiple classes with expected accuracy. Thus it appears the re-training pipleline should in theory work. This is the issue I resolved.

My next step is to just try using a larger training data set so it learns "not class 1" better, I suppose. Right now, I only have around 100 samples.

Have you made any progress?
